Ports/poppler/package.sh

```diff
@@ -7,36 +7,41 @@ files=(
     "https://poppler.freedesktop.org/poppler-data-${version_data}.tar.gz#c835b640a40ce357e1b83666aabd95edffa24ddddd49b8daff63adb851cdab74"
 )
 depends=(
+    'boost'
     'curl'
     'fontconfig'
     'freetype'
+    'gpgme'
+    'lcms2'
     'libjpeg'
     'libpng'
     'libtiff'
+    'openjpeg'
 )
 useconfigure='true'
 configopts=(
     '-B build'
     "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
     '-DCMAKE_BUILD_TYPE=release'
-    '-DENABLE_BOOST=OFF'
     '-DENABLE_CPP=OFF'
     '-DBUILD_CPP_TESTS=OFF'
     '-DENABLE_GLIB=OFF'
     '-DENABLE_GOBJECT_INTROSPECTION=OFF'
-    '-DENABLE_GPGME=OFF'
-    '-DENABLE_GTK_DOC=OFF'
-    '-DENABLE_LCMS=OFF'
-    '-DENABLE_LIBOPENJPEG=unmaintained'
+    '-DENABLE_GPGME=OFF' # Enabling GPGME causes the program to crash (#23557)
     '-DENABLE_NSS3=OFF'
     '-DENABLE_QT5=OFF'
     '-DBUILD_QT5_TESTS=OFF'
     '-DENABLE_QT6=OFF'
     '-DBUILD_QT6_TESTS=OFF'
     '-DBUILD_GTK_TESTS=OFF'
-    '-DBUILD_MANUAL_TESTS=OFF'   
+    '-DBUILD_MANUAL_TESTS=OFF'
 )
 
+pre_configure() {
+    # Fix the GpgmeppConfig.cmake file to use the correct path for cross-compilation
+    run sed -i'.bak' "s#/usr/local#${SERENITY_INSTALL_ROOT}/usr/local#g" ${SERENITY_INSTALL_ROOT}/usr/local/lib/cmake/Gpgmepp/GpgmeppConfig.cmake
+}
```

Comment:
I spend 2 days trying to and do now think that this is the correct behaviour. This file is simply not made for cross-compilation. It would work perfectly fine if it were used on system.

If somebody is able to do that without this hack, you are welcome to change it.